### DISCUSSION TITLE
Evolution Sheet

### DISCUSSION THREAD

**Discussion by LucasAlbuquerque0** [Telefonica/mistica-design#1564] [Proposals]
### Goal

Request the evolution of the Information Sheet to meet a need.

Pull Nestad Instance from Bullet to component
Add divide component between the information, possibility of controlling whether or not to use the divider
Add the action button will just be a "Close Action" it only works as a close, it does not generate action.

### Context

The need to evolve the component came from the Smart Home, the idea is to improve. Increasing the possibilities of using the sheet. It is very important to pull the bullet's nested instance out, making it easier to use not only the bullet but the icon.

### Impact

Evolution is an implementation of what already exists, avoiding rework to create new components. And it guarantees better personalization for the end user.

### Discussion Questions

Can we pull the Nestad Instance from Bullet and Icon to control the comp's props?

### Do you have a visual proposal?

https://www.figma.com/file/3FWICps6yc9do1FVBRxxgN/Bottom-Sheet-Evolu%C3%A7%C3%A3o?type=design&node-id=331%3A5823&mode=design&t=HIcff09r8lfVewmg-1

**Comment by yceballost** [comment]
good point! We are agree to keep behavior consistency between rowList and sheet rows! well spotted!
